modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCheck.java

```diff
@@ -97,6 +111,54 @@ private Map<String, String> _getMap(String fileName) throws Exception {
 		return map;
 	}
 
+	private boolean _isClassNameEquals(
+		String importName, String newImportName) {
+
+		_className = _getClassNameByImport(importName);
+		_newClassName = _getClassNameByImport(newImportName);
+
+		return _className.equals(_newClassName);
+	}
+
+	private String _replaceImpl(
+		String content, String className, String newClassName) {
+
+		String[] lines = content.split(StringPool.NEW_LINE);
+
+		for (String line : lines) {
+			if (line.contains(className) && !line.contains("import")) {
+				String newLine = StringUtil.replace(
+					line, className, newClassName);
+
+				content = StringUtil.replace(content, line, newLine);
+			}
+
+			String implClass = StringUtil.lowerCaseFirstLetter(className);
+
+			if (line.contains(implClass)) {
+				String newLine = StringUtil.replace(
+					line, implClass,
+					StringUtil.lowerCaseFirstLetter(newClassName));
+
+				content = StringUtil.replace(content, line, newLine);
+			}
+		}
+
+		return content;
+	}
+
+	private String _replaceImport(
+		String content, String importName, String newImportName) {
+
+		return StringUtil.replace(
+			content,
+			StringBundler.concat("import ", importName, StringPool.SEMICOLON),
+			StringBundler.concat(
+				"import ", newImportName, StringPool.SEMICOLON));
```

Comment:
Good job, @michaelmdelcavalcanti! You proactively applied the feedback from the last pull here. :clap: